lib/errorHandlers/suppressError.ts

```diff
@@ -15,25 +15,43 @@ const {
 
 const i18nKey = 'lib.errorHandlers.suppressErrors';
 
-function createPlatformVersionError(err, subCategory) {
+function createPlatformVersionError(err: unknown, subCategory: string): void {
   let translationKey = 'unspecifiedPlatformVersion';
   let platformVersion = 'unspecified platformVersion';
   const errorContext =
-    err.response && err.response.data && err.response.data.context;
+    typeof err === 'object' &&
```

Comment:
I think there are potentially a few problems with this. First - it looks like we always call `createPlatformVersionError` after confirming that an error is a `HubspotHttpError`, so we can update the type for `err`. Second, I think it might be possible that this code was never updated when we updated things to use `HubspotHttpError` - it looks `data` and `context` should exist on the error object

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1251/commits/146dcbafeabd6d865cb305cb61cf867207c31fbf